container.go

```diff
@@ -167,6 +167,15 @@ type ContainerRequest struct {
 	LogConsumerCfg          *LogConsumerConfig                         // define the configuration for the log producer and its log consumers to follow the logs
 }
 
+// sessionID returns the session ID for the container request.
+func (c *ContainerRequest) sessionID() string {
+	if sessionID := c.Labels[core.LabelSessionID]; sessionID != "" {
```

Comment:
If users pass their own sessionId value here, then the containers (and the networks) won't use the library's mechanism to determine the real sessionID (from parent go test process).

Is this intended?

Reply:
We checked that the user-defined labels are merged with the core labels, raising an error if the user-defined ones prepends the `org.testcontainers` prefix. We should notice that in the ContainerRequest struct's Labels field.